ff/serving/coordinator/coordinator.go

```diff
@@ -91,34 +114,30 @@ func NewCoordinator(meta *metadata.Client, logger *zap.SugaredLogger, cli *clien
 const MAX_ATTEMPTS = 1
 
 func (c *Coordinator) WatchForNewJobs() error {
+	c.Logger.Debug("Watching for new jobs")
 	getResp, err := (*c.KVClient).Get(context.Background(), "JOB_", clientv3.WithPrefix())
 	if err != nil {
 		return err
 	}
 	for _, kv := range getResp.Kvs {
-		fmt.Println("FOUND:", kv)
-		go func() {
+		go func(kv *mvccpb.KeyValue) {
 			err := c.executeJob(string(kv.Key))
 			if err != nil {
-				fmt.Println(err)
+				c.Logger.Errorf("Error executing job: %v", err)
 			}
-		}()
+		}(kv)
 	}
 	for {
-		fmt.Println("WATCHING")
 		rch := c.EtcdClient.Watch(context.Background(), "JOB_", clientv3.WithPrefix())
 		for wresp := range rch {
-			fmt.Println("FOUND:", wresp)
 			for _, ev := range wresp.Events {
-				fmt.Println("FOUND:", ev)
 				if ev.Type == 0 {
-					fmt.Println("EXECUTING:", ev.Kv.Key)
-					go func() {
+					go func(ev *clientv3.Event) {
 						err := c.executeJob(string(ev.Kv.Key))
 						if err != nil {
-							fmt.Println(err)
+							c.Logger.Errorf("Error executing job: %v", err)
```

Comment:
At whenever youre going through all the logging stuff, should come back to this and change it to be more structured. 
c.Logger.Debugw("Error executing job", "error", err)

Reply:
Makes it easier to search for errors if we add the logs into a indexer

Reply:
yes, I'll change it and for each level in the debugging heiarchy